File: Baekjoon/solved_ac/class2/1978.cpp
// 1978: 소수 찾기

#include <iostream>
using namespace std;

int main() {
  int N, prime=0;
  cin >> N;

  while (N--) {
    int num, cnt=0;
    cin >> num;

    for (int i=1; i<=num; i++) {
      if (num % i == 0) cnt++;
      if (cnt > 2) break;
    }

    if (cnt == 2) prime++;
  }
  cout << prime;

  return 0;
}

File: Baekjoon/solved_ac/class2/1978.py
# 1978: 소수 찾기

N = int(input())
nums = list(map(int, input().split()))

prime = 0
for num in nums:
  cnt = 0

  for i in range(1, num+1):
    if (num % i == 0): cnt += 1
    if (cnt > 2): break

  if (cnt == 2): prime += 1

print(prime)